client/src/styles/assignRoles.css

```diff
@@ -0,0 +1,20 @@
+.container {
```

Comment:
Can this file be removed and the styles moved into a makeStyles function?

Reply:
I have deleted this .css file and moved the styles to makeStyles function.